packages/language-support/src/tests/consoleCommands.test.ts

```diff
@@ -93,11 +97,82 @@ describe('sanity checks', () => {
         tokenType: 'consoleCommand',
       },
     ]);
+    expect(applySyntaxColouring(':connect')).toEqual([
+      {
+        length: 1,
+        position: {
+          line: 0,
+          startCharacter: 0,
+          startOffset: 0,
+        },
+        token: ':',
+        tokenType: 'consoleCommand',
+      },
+      {
+        length: 7,
+        position: {
+          line: 0,
+          startCharacter: 1,
+          startOffset: 1,
+        },
+        token: 'connect',
+        tokenType: 'consoleCommand',
+      },
+    ]);
+    expect(applySyntaxColouring(':disconnect')).toEqual([
+      {
+        length: 1,
+        position: {
+          line: 0,
+          startCharacter: 0,
+          startOffset: 0,
+        },
+        token: ':',
+        tokenType: 'consoleCommand',
+      },
+      {
+        length: 10,
+        position: {
+          line: 0,
+          startCharacter: 1,
+          startOffset: 1,
+        },
+        token: 'disconnect',
+        tokenType: 'consoleCommand',
+      },
+    ]);
+    expect(applySyntaxColouring(':sysinfo')).toEqual([
+      {
```

Comment:
Given `server` is the only especial command (at colouring level), could you please add a test for that?

Reply:
I've got a  `:server` suite below, which includes some syntax colouring tests, are these covering what you're asking?